tools/pot/openvino/tools/pot/algorithms/quantization/fake_quantize_configuration.py

```diff
@@ -290,6 +290,15 @@ def _get_unified_scales_ops(hw_ops_):
                 unified_scales_ops_.append(hw_op)
         return unified_scales_ops_
 
+    def _get_quantizable_per_ch_ops(hw_cfg_):
```

Comment:
A graph test for the use case is required here.

Reply:
I've added test into `test_unify_scales.py`. The graph tests check only FQ insertion and doesn't call FQ unification algo. I'll discuss test coverage with @KodiaqQ